pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1271,6 +1272,22 @@ def test_plot_outofbounds_datetime(self):
         values = [datetime(1677, 1, 1, 12), datetime(1677, 1, 2, 12)]
         self.plt.plot(values)
 
+    def test_format_timedelta_ticks(self):
+        import matplotlib.pyplot as plt
+        rng = timedelta_range('0', periods=3, freq='ns')
+        df = DataFrame(np.random.randn(len(rng), 3), rng)
+        ax = df.plot()
+        plt.gcf().autofmt_xdate()
+        xaxis = ax.get_xaxis()
+        for l in xaxis.get_ticklabels():
+            if len(l.get_text()) > 0:
+                self.assertEqual(l.get_rotation(), 30)
+
+    def test_timedelta_plot(self):
+        # test issue #8711
+        s = Series(range(5), timedelta_range('1day', periods=5))
+        _check_plot_works(s.plot)
+
```

Comment:
This basically just tests that the plot call does not error. So, for example, that means that it will test that it does not raise a MemoryError like it does now in some cases